docs/content/dev/release_checklist.md

```diff
@@ -0,0 +1,195 @@
+  +      r_lang = requests.post('https://api.poeditor.com/v2/projects/export', dict(api_token=project_api, id=project_id, language=lang['code'], type="key_value_json"))
+  ```
+  2. run the script.
+  3. update the json files in the `locales` directory.
+    - any languages with 0% translations should not be included
+  4. If any languages are new, they need to be added to `locales/_supported.json`
+- [ ] Write release notes
+  - [ ] Update date
+  - [ ] General description
+  - [ ] Things requiring special action beside updating the software
+  - [ ] New features
+  - [ ] Bug fixes
+  - [ ] Enhancements
+  - [ ] Add all contributors
+    - sort alphabetically
+      - [ ]Update `AUTHORS` file
+
+## Final Testing:
+
+- [ ] Create release tar ball
+  - `mkdir /tmp/hedgedoc && cd /tmp/hedgedoc`
+  - `git clone -b master https://github.com/hedgedoc/hedgedoc.git .`
+  - `yarn install`
+  - `yarn build`
+  - `cd ..`
+  - `tar cvzf hedgedoc-x.x.x.tar.gz --sort=name --exclude hedgedoc/node_modules --exclude hedgedoc/.git --exclude hedgedoc/.github --exclude hedgedoc/.yarn/cache hedgedoc`
+
+Use this tar ball to test the following things:
+
+### Account system
+
+- [ ] User registration works
+- [ ] User login works
+- [ ] User self-deletion works
+
+### Notes
+
+- [ ] Create new note works
+- [ ] Create new note with custom alias works when FreeURL-mode is enabled
+- [ ] Create new note with custom alias fails when FreeURL-mode is disabled
+- [ ] New note keeps content (visit, write something, leave, visit again after a minute)
+- [ ] API `POST /new` works
+  - `curl -i -d '# hello world' -H "Content-Type: text/markdown" http://localhost:3000/new`
+- [ ] API `POST /new/some-test-note` works when FreeURL-mode is enabled
+- [ ] API `POST /new/some-test-note` fails when FreeURL-mode is disabled
+- [ ] Working with 2 (or more) devices on a page works and results in the same document
+- [ ] Uploads work for images
+- [ ] Uploads fail for other data (e.g. binaries)
+
+### Database
+
+- [ ] No problems when starting with
+  - [ ] SQLite
+  - [ ] Postgres
+    - `docker run -d --name=hd1-pg -p 5432:5432 -e POSTGRES_USER=hd1db -e POSTGRES_PASSWORD=hd1db -e POSTGRES_DB=hd1db postgres:latest`
+    - config:
+```
+"db": {
+  "username": "hd1db",
+  "password": "hd1db",
+  "database": "hd1db",
+  "host": "localhost",
+  "port": "5432",
+  "dialect": "postgres"
+},
+```
+    
+  - [ ] MariaDB
+    - `docker run --name=hd1-mysql -p 3306:3306 -e MARIADB_USER=hd1db --env MARIADB_PASSWORD=hd1db --env MARIADB_DATABASE=hd1db -e MARIADB_RANDOM_ROOT_PASSWORD=true --rm -d mariadb:latest`
+    - config:
+```
+"db": {
+  "username": "hd1db",
+  "password": "hd1db",
+  "database": "hd1db",
+  "host": "localhost",
+  "port": "5432",
+  "dialect": "mariadb"
+},
+```
+- [ ] Keeps content of already existing SQLite file from older version
```

Comment:
This section breaks the checkboxes because of the config snippets.

I would rather make a subheading per database and add a checkbox and the explanations there.